README.txt

```diff
@@ -6,16 +6,16 @@ Tested up to: 4.5.6
 License: MIT License, GNU General Public License v2.0
 License URI: MIT-LICENSE.txt, GPL-LICENSE.txt
 
-Algolia Search plugin for WordPress is a drop in replacement for WordPress search. It provides a find-as-you-type search results page and adds an autocomplete dropdown menu to your search bars.
+Algolia Search plugin for WordPress is a drop in replacement for WordPress search. It provides a find-as-you-type search results page and adds an autocomplete dropdown menu to your search bar.
```

Comment:
Plural was intentional because it can actually power multiple search bars on the same page. Not sure it is necessary to mention it though.